lib/fog/vsphere/requests/compute/vm_clone.rb

```diff
@@ -838,6 +838,7 @@ def modify_template_volumes_specs(vm_mob_ref, volumes)
               end
               template_volume.backing.diskMode = new_volume.mode
               template_volume.backing.thinProvisioned = new_volume.thin
+              template_volume.backing.eagerlyScrub = new_volume.eager_zero
```

Comment:
Oh, we cannot have `thin` and `eager_zero` at the same time. Last thing then, sorry for not noticing before:
```suggestion
              template_volume.backing.eagerlyScrub = !new_volume.thin && new_volume.eager_zero
```